files/KoreBuild/scripts/commands/docker/Ubuntu.dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:16.04
+
+ARG BUILD_ARGS
+
+RUN apt-get update && \
```

Comment:
Does this need to be Ubuntu 16? Or can we use existing `microsoft/dotnet` images that already install most of this?

e.g.: https://github.com/dotnet/dotnet-docker/blob/master/2.0/runtime-deps/stretch/Dockerfile